### PR TITLE
Modify the faq.md in the English version

### PR DESCRIPTION
**Pull request by littletiger123**
### Ⅰ. Describe what this PR does
This PR checks the spelling and grammar in faq.md in the English version.

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: docs/en/userguide/faq.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ If the csidriver object exists, please check if the csi registered node contains
 ```
 kubectl get csinode | grep <node_name>
 ```
-If the above command has no output, check whether the kubelet configuration of the node which the task is scheduled on is the default value `/var/lib/kubelet`. Because Fluid's CSI component is registered to kubelet through a fixed address socket, the default value is `--csi-address=/var/lib/kubelet/csi-plugins/fuse.csi.fluid.io/csi.sock --kubelet-registration-path=/var/lib/kubelet/csi-plugins/fuse.csi.fluid.io/csi.sock`.
+If the above command has no output, check whether the kubelet configuration of the node to which the task is scheduled is the default value `/var/lib/kubelet`. Because Fluid's CSI component is registered to kubelet through a fixed address socket, the default value is `--csi-address=/var/lib/kubelet/csi-plugins/fuse.csi.fluid.io/csi.sock --kubelet-registration-path=/var/lib/kubelet/csi-plugins/fuse.csi.fluid.io/csi.sock`.
```
Comment (jasonliu747) [inline docs/en/userguide/faq.md:98]: on which

File: docs/en/userguide/faq.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ You can recreate the dataset, and the new dataset will display these fields norm
 
 ## 6. Why do I run the example [Nonroot access](../samples/nonroot_access.md), and I  encounter mkdir permission denied error
 
-**Answer**: In nonroot scenirio,Firstly,you have to check that you pass the right useinfo to runtime,Secondly,you should check the alluxio master pod status,and use journalctl to see the kubelet logs in the node of alluxio master pod.the mkdir error was cause when mount the hostpath to container,So we have to check the root have right permission to exec the directory.We can use stat to check if root can exec the directory.For example in the below root have permission to operator /dir
+**Answer**: In nonroot scenario, Firstly, you have to check that you pass the right useinfo to runtime, Secondly, you should check the alluxio master pod status, and use journalctl to see the kubelet logs in the node of alluxio master pod. The mkdir error was caused when mounting the hostpath to the container, So we have to check the root has the right permission to exec the directory. We can use stat to check if root can exec the directory. For example in the below root have permission to operator /dir
```
Comment (jasonliu747) [inline docs/en/userguide/faq.md:110]: In non-root scenario, you have to check if you pass the right user info(?) to runtime first. Secondly, you should check the alluxio master pod status, and use journalctl to see the kubelet logs in the node of alluxio master pod.  The mkdir error was caused when mounting the hostpath to the container and therefor we have to check the root has the right permission to exec the directory.

**Review by jasonliu747 — CHANGES_REQUESTED** [review]

File: docs/en/userguide/faq.md (hunk 2 above)
Comment (littletiger123) [reply docs/en/userguide/faq.md]: Fixed, thanks.

File: docs/en/userguide/faq.md [hunk 3]
```diff
@@ -107,7 +107,10 @@ You can recreate the dataset, and the new dataset will display these fields norm
 
 ## 6. Why do I run the example [Nonroot access](../samples/nonroot_access.md), and I  encounter mkdir permission denied error
 
-**Answer**: In nonroot scenirio,Firstly,you have to check that you pass the right useinfo to runtime,Secondly,you should check the alluxio master pod status,and use journalctl to see the kubelet logs in the node of alluxio master pod.the mkdir error was cause when mount the hostpath to container,So we have to check the root have right permission to exec the directory.We can use stat to check if root can exec the directory.For example in the below root have permission to operator /dir
+**Answer**: In non-root scenario, you have to check if you pass the right user info to runtime first. Secondly, you should 
+check the alluxio master pod status, and use journalctl to see the kubelet logs in the node of alluxio master pod. The mkdir 
+error was caused when mounting the hostpath to the container and therefor we have to check the root has the right permission to exec the directory. 
+For example in the below root have permission to operator /dir
```
Comment (jasonliu747) [inline docs/en/userguide/faq.md:113]: Maybe keep this in single line instead of four?

**Review by jasonliu747 — COMMENTED** [review]
One tiny suggestion, others LGTM

**Review by jasonliu747 — APPROVED** [review]
/lgtm

**Pull request merged**